packages/wells/src/client/api/wells.ts

```diff
@@ -143,22 +144,29 @@ export class Wells extends AssetsAPI {
    * @param limit max number of objects to be returned
    * @param offset the starting offset of objects in the results
    */
-  public getWellsByPolygon = async (
-    polygon: string,
-    source: string,
-    crs: string,
-    layerName: string,
-    limit: number = 1000,
-    offset: number = 0
-  ): Promise<Well[]> => {
+  public getWellsByPolygon = async ({
+    polygon,
+    source = 'wellmodel',
+    crs = 'epsg:4326',
+    layerName = 'point',
+    limit = 1000,
+    offset = 0,
+  }: {
+    polygon: string;
+    source?: string;
+    crs?: string;
+    layerName?: string;
+    limit?: number;
+    offset?: number;
+  }): Promise<Well[]> => {
```

Comment:
is this how you wanted it? @rek

Reply:
now you are able to do like: `client.wells.getWellsByPolygon({polygon: geometry})` where `geometry = 'POLYGON ((-4.86423 63.59999, 19.86423 63.59999, 19.86423 52.59999, -4.86423 52.59999, -4.86423 63.59999))'`

Reply:
it would be nice if you handled the WKT transformation so we can just send GeoJSON, that way when @kiril-me supports that from `geospatial-sdk-js` no changes are required.

you could also detect a string (wkt) or an object (geojson) and set the query based on that.

also it's still missing a few fields we need. i put up a quick PR here to should how discover would use it: https://github.com/cognitedata/discover/pull/718/files